src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
The 'main' method should not begin with an empty line. Please remove the redundant empty line at the beginning of the method body.